src/invert.test.ts

```diff
@@ -0,0 +1,55 @@
+import { invert } from './invert';
+import { pipe } from './pipe';
+
+describe('invert', function () {
+  describe('with parameter', () => {
```

Comment:
I think the convention is calling these "data-first" and "data-last"... it wasn't clear to me when reading this what parameter you are referring to

Reply:
You're right. I'll change them.